packages/lesswrong/server/callbacks/userCallbacks.ts

```diff
@@ -195,3 +198,65 @@ getCollectionHooks("Users").editSync.add(async function usersEditCheckEmail (mod
   }
   return modifier;
 });
+
+getCollectionHooks("Users").editAsync.add(async function subscribeToForumDigest (newUser: DbUser, oldUser: DbUser) {
+  if (
+    isAnyTest ||
+    forumTypeSetting.get() !== 'EAForum' ||
+    newUser.subscribedToDigest === oldUser.subscribedToDigest
+  ) {
+    return;
```

Comment:
Can you tell me how you've been testing?

Reply:
Sure! I've temporarily added the Mailchimp keys to devDbSettings, and I've tested by simply going through the signup flow (+ toggling via the account settings page) locally.

Reply:
And disabled the test for isAnyTest presumably